Samples/SensorActivityMonitorConsoleApp/SensorActivityMonitorConsoleApp/cameraActivityCallback.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#pragma once
+
+class MyCameraNotificationCallback : public winrt::implements <MyCameraNotificationCallback, IMFSensorActivitiesReportCallback>

```

Comment:
>implements [](http://example.com/codeflow?start=51&length=10)

can we use WRL instead of mix and match of wil and winrt?

Reply:
How's that different than mix and match wil and WRL? WRL also has ComPtr, okay it's more interoperable between WRL::ComPtr and wil::com_ptr.

c++/winrt is the recommended way over WRL

I can create CreateInstance methdod that will return wil::com_ptr or so that we do not need mix winrt::com_ptr and wil::com_ptr elsewhere.

Reply:
most of the winrt stuff throws exceptions so needs to be under try catch. 
That way we get uniformity. 
BTW is there a wil equivalent for "implements"
and also not to confuse the user with winrt while demonstrating a win32 api. (external developers do get confused with the word winrt.)

Reply:
Added comment with link to documentation that this is only a way to implement classic COM.

And CreateInstance methods contain all the winrt stuff so for class user it's just COM.